docker/docker-compose.yaml

```diff
@@ -0,0 +1,170 @@
+version: '3'
```

Comment:
We could use '2'. It's mostly to encourage using the current version of things. Looking at https://docs.docker.com/compose/compose-file/compose-versioning/#version-2x-to-3x I didn't use anything from 3, but 2 is a minimum for the networks and services.